src/scheduler/agent/database.c

```diff
@@ -1132,6 +1118,30 @@ char* get_email_command(scheduler_t* scheduler, char* user_email)
         g_string_append(client_cmd, "warn");
       }
     }
+    if(g_hash_table_contains(smtpvariables, "SMTPAuthUser") && g_hash_table_contains(smtpvariables, "SMTPAuthPasswd")
+          && g_hash_table_contains(smtpvariables, "SMTPHostName") && g_hash_table_contains(smtpvariables, "SMTPPort"))
+    {
+      if(system("s-nail --help") == 0)
```

Comment:
It won't be required to check if `s-nail` is installed or not. As with all the OS supported by FOSSology at the moment, it is available. So you can directly modify this value to point to `s-nail` https://github.com/fossology/fossology/blob/45e186a68fc3c3fbf42dad0a8e92dce3e39922f7/install/defconf/fossology.conf.in#L92

Reply:
Also, you can remove `heirloom-mailx` from here: https://github.com/fossology/fossology/blob/45e186a68fc3c3fbf42dad0a8e92dce3e39922f7/debian/control#L82